sdkconfig.defaults

```diff
@@ -82,9 +85,9 @@ CONFIG_PARTITION_TABLE_SINGLE_APP=
 CONFIG_PARTITION_TABLE_TWO_OTA=
 CONFIG_PARTITION_TABLE_CUSTOM=y
 CONFIG_PARTITION_TABLE_CUSTOM_FILENAME="partitions-16MB.csv"
-CONFIG_PARTITION_TABLE_CUSTOM_APP_BIN_OFFSET=0x10000
 CONFIG_PARTITION_TABLE_FILENAME="partitions-16MB.csv"
-CONFIG_APP_OFFSET=0x10000
+CONFIG_PARTITION_TABLE_OFFSET=0x8000
```

Comment:
Dit is waarschijnlijk waar ik eerder steeds op borkte

Reply:
geen idee; ik zou verwachten dat je deze setting default meekrijgt.